src/background/controller/wallet.ts

```diff
@@ -124,8 +124,14 @@ export class WalletController extends BaseController {
 
   /* wallet */
   boot = async (password) => {
-    const result = await keyringService.boot(password);
-    return result;
+    const isBooted = await keyringService.isBooted();
+    console.log('boot', password, await keyringService.isBooted());
```

Comment:
Remove those log in production. I think we should have debugPrint func, which will only print in console in not production environment.